pyFV3/stencils/ray_fast.py

```diff
@@ -36,7 +37,7 @@ def compute_rf_vals(pfull, bdt, rf_cutoff, tau0, ptop):
 @gtscript.function
 def compute_rff_vals(pfull, dt, rf_cutoff, tau0, ptop):
     rffvals = compute_rf_vals(pfull, dt, rf_cutoff, tau0, ptop)
-    rffvals = 1.0 / (1.0 + rffvals)
+    rffvals = f64(1.0) / (f64(1.0) + rffvals)
```

Comment:
In an f32 run, aren't you only promoting the constants here and not rffvals?

Reply:
The upcaster will force in the `rffvals` to be f64. Alternatively you could, type hint `rffvals` e.g.
```python
rffvals: f64 = ...
```

Reply:
The `upcaster` follows basic C up-casting rules